targets/nintendoswitch.s

```diff
@@ -0,0 +1,50 @@
+.section .text.jmp, "x"
+.global _start
+_start:
+        b start
+	.word _mod_header - _start
```

Comment:
This file uses a mix of tabs and spaces. Can you format it to use one or the other?
(My preference is spaces but that doesn't matter much, as long as it's consistent).